types/react-transition-group/CSSTransition.d.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from "react";
+import { TransitionActions } from "react-transition-group";
+import { TransitionProps } from "react-transition-group/Transition";
+
+export interface CSSTransitionClassNames {
+    appear?: string;
+    appearActive?: string;
+    enter?: string;
+    enterActive?: string;
+    exit?: string;
+    exitActive?: string;
+}
+
+export interface CSSTransitionProps extends TransitionProps {
+    classNames: string | CSSTransitionClassNames;
```

Comment:
It's unnecessary for the type checking itself, but very useful in combination with tsservice (vscode autocompletion, etc)